src/tdhLoop/tdh.ts

```
@@ -139,12 +112,24 @@ export function getDefaultBoost(): DefaultBoost {
       acquired_info: []
     },
     gradients: {
-      available: 0.06,
-      available_info: ['0.02 for each Gradient up to 3'],
+      available: 0.10,
```

ESLint

[error] Delete `0`

(prettier/prettier)